paasta_tools/paasta_metastatus.py

```diff
@@ -27,32 +27,25 @@
 from marathon.exceptions import MarathonError
 
 from paasta_tools import __version__
-from paasta_tools import adhoc_tools
-from paasta_tools import chronos_tools
-from paasta_tools import kubernetes_tools
-from paasta_tools import marathon_tools
-from paasta_tools import tron_tools
 from paasta_tools.autoscaling.autoscaling_cluster_lib import AutoscalingInfo
 from paasta_tools.autoscaling.autoscaling_cluster_lib import get_autoscaling_info_for_all_resources
 from paasta_tools.chronos_tools import get_chronos_client
 from paasta_tools.chronos_tools import load_chronos_config
+from paasta_tools.cli.utils import get_instance_config
 from paasta_tools.marathon_tools import get_marathon_clients
 from paasta_tools.marathon_tools import get_marathon_servers
 from paasta_tools.mesos.exceptions import MasterNotAvailableException
 from paasta_tools.mesos.master import MesosState
 from paasta_tools.mesos_tools import get_mesos_master
 from paasta_tools.metrics import metastatus_lib
-from paasta_tools.paasta_service_config_loader import PaastaServiceConfigLoader
-from paasta_tools.utils import DEFAULT_SOA_DIR
 from paasta_tools.utils import format_table
 from paasta_tools.utils import load_system_paasta_config
 from paasta_tools.utils import paasta_print
 from paasta_tools.utils import PaastaColors
 from paasta_tools.utils import print_with_indent
-from paasta_tools.utils import validate_service_instance
 
 
-log = logging.getLogger(__name__)
+log = logging.getLogger('paasta_metastatus')
```

Comment:
Why?

Reply:
When you actually see an error it would say ERROR.__main__ which I thought wasn't that helpful so I put in the filename.